bin/pact-cli.ts

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+import pact from "../src/pact";
+import cli = require("caporal");
+
+const pkg = require("../package.json");
+
+cli
+	.name("pact")
+	.bin("pact")
+	.description("Pact CLI tool")
+	.version(pkg.version);
+
+cli
+	.command("mock")
+	.description("Creates a pact mock service to start contract testing")
+	.option("-p, --port [n]", "Port on which to run the service. Default is random.", cli.INT)
+	.option("-h, --host [hostname]", "Host on which to bind the service. Default is localhost.")
+	.option("-l, --log [file]", "File to which to log output to.")
+	.option("-s, --ssl [boolean]", "Use a self-signed SSL cert to run the service over HTTPS. Default is false (HTTP).", cli.BOOL)
+	.option("-o, --cors [boolean]", "Support browser security in tests by responding to OPTIONS requests and adding CORS headers to mocked responses. Default is false.", cli.BOOL)
+	.option("-d, --pact-dir [directory]", "Directory to which the pacts will be written. Default is cwd.")
+	.option("-i, --pact-version [n]", "The Pact specification version to use when writing the Pact files. Default is 1.", cli.INT)
+	.option("--consumer [consumerName]", "Specify consumer name for written Pact files.")
+	.option("--provider [providerName]", "Specify provider name for written Pact files.")
+	.action((args, options) => pact.createServer(options).start());
+
+cli
+	.command("verify")
+	.description("Verifies Pact Contracts on the current provider")
+	.option("-b, --provider-base-url <URL>", "The Pact Provider base URL.")
+	.option("-u, --pact-urls [URLs]", "Comma separated list of Pact files or URIs.", cli.LIST)
+	.option("-p, --provider [name]", "Name of the Provider to verify. Required if not using --pact-urls.")
+	.option("-bu, --pact-broker-url [URL]", "URL of the Pact Broker to retrieve pacts from. Required if not using --pact-urls.")
+	.option("-pu, --provider-states-url [URL]", "URL to fetch provider states from.")
```

Comment:
I reckon we should drop this flag, it was deprecated and given this will likely be a major release it feels like a good time to make it disappear

Reply:
Which flag specifically? the provider states one?

Reply:
ah, the provider-states-url one.  Removing.